draft-jones-perc-dtls-tunnel/draft-jones-perc-dtls-tunnel.md

```diff
@@ -496,7 +508,7 @@ struct {
 ```
 
 The fields are described as follows:
-* association_id: The value of the association identifier used to uniquely identify each endpoint in a conference
+* association_id: The value of the association identifier used to uniquely identify each endpoint connected to the media distributor
```

Comment:
Yeah.  I accepted the PR and then changed all three of the association_id: definitions to be consistent and along the lines of what I wanted.  See this commit: https://github.com/ietf/perc-wg/commit/30706a8830587f4575bae2fa5273915be959eec6.  Feel free to suggest more changes.